addons/longhorn/template/base/install.sh

```diff
@@ -188,11 +192,38 @@ function maybe_init_hosts() {
 }
 
 # if rook-ceph is installed but is not specified in the kURL spec, migrate data from rook-ceph to longhorn
-function maybe_migrate_from_rook() {
+function longhorn_maybe_migrate_from_rook() {
     if [ -z "$ROOK_VERSION" ]; then
         if kubectl get ns | grep -q rook-ceph; then
             rook_ceph_to_longhorn
             export DID_MIGRATE_ROOK_PVCS="1" # used to automatically delete rook-ceph if object store data was also migrated
         fi
     fi
 }
+
+function longhorn_bail_upgrade_unsupported() {
+    local current_version="$(longhorn_current_version)"
+    semverParse "${current_version}"
+    local current_version_minor="${minor}"
+
+    if [ -z "$current_version" ]; then
+        return
+    fi
+
+    semverParse "${LONGHORN_VERSION}"
+    local next_version_minor="${minor}"
+
+    local previous_version_minor="$((next_version_minor-1))"
+
+    if [ "$current_version_minor" -lt "$previous_version_minor" ]; then
+        logFail "Upgrades to Longhorn version ${LONGHORN_VERSION} from versions prior to 1.${previous_version_minor}.x are unsupported."
+        bail "Please upgrade to the previous minor version first."
+    fi
```

Comment:
I'm genuinely unsure - is this the behavior we want? For instance, in rook, we just skip the upgrade and continue on with the rest of the kurl install instead of exiting with an error

Reply:
i will make the behavior mimic rook